app/assets/stylesheets/earthworks.css

```diff
@@ -164,3 +164,7 @@ Uncomment lines 142-144
   --bs-pagination-focus-bg: none;
   --bs-pagination-focus-box-shadow: none;
 }
+
+.document-metadata dt, .document-metadata dd {
```

Comment:
Are you trying to remove the `.dl-invert` styles here? If so, can you init your DocumentMetadataComponent without that class? https://github.com/projectblacklight/blacklight/blob/a8b5a3d3a2e8243c8a875d40528eae5c106d83bd/app/components/blacklight/document_metadata_component.rb#L12

Reply:
not remove, just add text-start to those, but yes I would want DocumentMetadataComponent to have another class added.

Reply:
I love additive styles rather than subtractive ones.

Reply:
I think that works. I wasn't sure if really made sense to do a component override for such a small thing that could be fixed with css.

Reply:
@jcoyne .dl-invert has more css than just text-align. What if we override the class instead?

Reply:
The color? https://github.com/projectblacklight/blacklight/blob/4934e604e4538d22afff635579c8d11f9e23aba4/app/assets/stylesheets/blacklight/_balanced_list.scss#L4.   Ya'll can do what you want, but when maintaining an app, I find that CSS overrides are very hard to understand what they are trying to do compared to just applying classes that add style.